src/planar_line_renderer.rs

```diff
@@ -92,6 +95,11 @@ impl PlanarLineRenderer {
             colors.clear()
         }
     }
+
+    /// Sets the line width for the rendered lines.
+    pub fn set_line_width(&mut self, line_width: f32) {
```

Comment:
Can we assert strictly positive `line_width` (and document that) to avoid a cryptic panic downstream?